{{cookiecutter.project_name}}/requirements.txt

```diff
@@ -1,4 +1,4 @@
-git+https://github.com/mlflow/mlflow.git@4b259e
+mlflow>=2.0
```

Comment:
@smurching do we want it pinned at 2.0 or set to 2.0 or higher?

Reply:
We might want to pin at 2.0 - the recipes API is still experimental so customers may see breakages with new versions. Unfortunately pinning 2.0 is also a pretty restrictive requirement on customers' local development environment, but seems like the safer option for now

Reply:
Makes sense to me, @sunishsheth2009 can we have this set to `mlflow==2.0`